app/components/forms/admin/settings/payment-gateway-form.js

```diff
@@ -152,6 +175,12 @@ export default Component.extend(FormMixin, {
             'paypalClient'        : null
           });
         }
+        if (this.isCheckedAliPay === false) {
```

Comment:
no point in comparing something to false explicitly, use !.

Reply:
@CosmicCoder96 Done.